wp-content/themes/csisnuclearnetwork/assets/_scss/pages/post.scss

```diff
@@ -133,10 +133,58 @@
 
 .post {
   &__authors {
-    margin-top: rem(48);
+    margin-top: rem(24);
     margin-bottom: rem(32);
```

Comment:
This changes to `48` on larger screens.  I think the large breakpoint with no space between the author and cite this page is an error, but please ask Christina.

Reply:
Resolved